README.md

```diff
@@ -10,6 +10,12 @@ Most of the functions have its own tests and examples in the JS Doc.
 
 Feel free to either contribute yourself or submit an issue if there is a bug or you have an idea for a new extension.
 
+# Why to choose ramda-extension over other libraries ?
+
+We are trying to focus on functional point-free code. Our goal is to keep codebase composed only from ramda functions, not from own code. Basically, we are using only `const` and `ES6 modules` from the JavaScript itself.
+
+We have a few functions with side-effects (like `log` and `trace`) to support better developer's experience in debugging process. It is only for development purpose and it should not be used in production code.
```

Comment:
i am no english god.. but "like" to "such as"? 👯‍♂️

Reply:
https://english.stackexchange.com/questions/129976/difference-between-such-as-and-like